test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -156,6 +157,40 @@ public void AutoFitRankingTest()
                 Assert.True(col.Name == expectedOutputNames[col.Index]);
         }
 
+        [LightGBMFact]
+        public void AutoFitRankingCVTest()
+        {
+            string labelColumnName = "Label";
+            string groupIdColumnName = "GroupIdCustom";
+            string featuresColumnVectorNameA = "FeatureVectorA";
```

Comment:
**Suggestion** As I side note, I've looked into PR #5246 (which is already merged) and wanted to make the following minor suggestion:
Wherever the string `"GroupId"` was hardcoded, can you please change it to be `DefaultColumnNames.GroupId`. For example, in here:
[https://github.com/dotnet/machinelearning/blob/5d0edf24999290729659e4b7c701ab962ab59a17/src/Microsoft.ML.AutoML/API/RankingExperiment.cs#L37](https://github.com/dotnet/machinelearning/blob/5d0edf24999290729659e4b7c701ab962ab59a17/src/Microsoft.ML.AutoML/API/RankingExperiment.cs#L37)

I think it only appears 3 times on that file.

Also, while you're at it, can you please also change it on the default for the RankingEvaluator here:
[https://github.com/antoniovs1029/machinelearning/blob/bb13d629000c218136e741b643767cf45ae12fc4/src/Microsoft.ML.Data/Evaluators/RankingEvaluator.cs#L66](https://github.com/antoniovs1029/machinelearning/blob/bb13d629000c218136e741b643767cf45ae12fc4/src/Microsoft.ML.Data/Evaluators/RankingEvaluator.cs#L66)

It seems those 4 places are the only ones that would need to be updated (note: the `"GroupId"` string also appears on the SvmLoader, but I think it's better to not update those). Thanks :)